Resources/public/templates/partials/tree.hbt

```diff
@@ -1,6 +1,6 @@
 <ul class="ez-tree-level">
 {{#each children}}
-    <li class="ez-tree-node{{#if state.leaf}} is-tree-node-leaf{{/if}} {{#if state.open}}is-tree-node-open{{else}}is-tree-node-close{{/if}}" data-node-id="{{id}}">
+    <li class="ez-tree-node{{#if state.leaf}} is-tree-node-leaf{{/if}} {{#if state.open}}is-tree-node-open{{else}}is-tree-node-close{{/if}}" data-node-id="{{id}}" title="{{ data.contentInfo.name }}">
```

Comment:
IMO this is a good example of searching by actual text being useful / correct.

In order to use the location ids, AFAICT, there are only 2 options: 
1: Use ids instead of names in sentences (which doesn't make much sense), or 
2: Use the API to retrieve ids from contents when performing tests (`When` / `Then` steps).

The problem with the second approach is that using the API introduces additional complexity as it's not solely using/testing the application interface, and could potentially add points of failure (caching issues between API and UI, for example).